src/mpb.cpp

```diff
@@ -634,8 +634,8 @@ void fields::get_eigenmode_coefficients(dft_flux flux,
   if (d==NO_DIRECTION)
    abort("cannot determine normal direction in get_eigenmode_coefficients");
 
-  if (S.multiplicity() > 1)
-   abort("symmetries are not yet supported in get_eigenmode_coefficients");
+  //if (S.multiplicity() > 1)
```

Comment:
Can we save the value of `use_symmetry` in the `dft_flux` object and still abort `if (use_symmetry && S.multiplicity() > 1)`, i.e. in case an ordinary `add_dft_flux` was used?